lib/pdf/reader/width_calculator/composite.rb

```diff
@@ -22,7 +22,11 @@ def glyph_width(code_point)
 
         w = @widths[code_point]
         # 0 is a valid width
-        return w.to_f unless w.nil?
+        if w
+          w.to_f
+        else
+          0
+        end
       end
```

Comment:
This method was sometimes returning a `Numeric`, sometimes returning `nil`. Oops, let's consistently return `Numeric`.